pr-dhl-woocommerce/pr-dhl-woocommerce.php

```diff
@@ -705,28 +720,52 @@ public function is_shipping_domestic( $country_receiver ) {
 	/**
 	 * Function return whether the sender and receiver country is "crossborder" i.e. needs CUSTOMS declarations (outside EU)
 	 */
-	public function is_crossborder_shipment( $country_receiver ) {
+	public function is_crossborder_shipment( $shipping_address ) {
+        $is_crossborder = true;
 
-		$is_crossborder = false;
-		if ($this->is_shipping_domestic( $country_receiver )) {
-			$is_crossborder = false;
-		} else {
-			// Is sender country in EU...
-			if ( in_array( $this->base_country_code, $this->eu_iso2 ) ) {
-				// ... and receiver country is in EU means NOT crossborder!
-				if ( in_array( $country_receiver, $this->eu_iso2 ) ) {
-					$is_crossborder = false;
-				} else {
-					$is_crossborder = true;
-				}
-			} else {
-				$is_crossborder = true;
+		if ($this->is_shipping_domestic( $shipping_address['country'] )) {
+            $is_crossborder = false;
+		}
+
+        $base_address = [
+                'country' => $this->base_country_code,
+                'postcode'      => $this->get_base_postcode()
```

Comment:
```suggestion
                'country'  => $this->base_country_code,
                'postcode' => $this->get_base_postcode()
```
Another space suggestion.